.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -8,33 +8,30 @@ body:
   - type: markdown
     attributes:
       value: |
-        ## Friendly Reminder
-        As a team with very limited development time, we require that all issues be described accurately and concisely.
-        Failure to adhere to these guidelines will result in your issue being closed without response.
+        ## Reminder
+        To resolve your problem as soon as possible, please describe your problem as accurate and concise.
   
-        **Feature Requests:** Do not submit them here. Use [Discussions](https://github.com/Floorp-Projects/Floorp/discussions/new?category=feature-request).
+        **Feature Request:** Use [Discussions](https://github.com/Floorp-Projects/Floorp/discussions/new?category=feature-request) instead of GitHub Issues.
         
         For Japanese: 日本人の場合は、日本語で書いても構いません。
   
         ## Disclaimer
-        **Please note:**
-        - No support for operating systems below Windows 10.
-        - DRM content is not supported. Do not report DRM-related issues.
+        > [!NOTE]
+        > Windows 7, 8, 8.1, and below is not supported because of maintaining cost.
+        > DRM is partly supported currently.
+        > Moreover, kindly refer [Issue Trackers.](https://docs.floorp.app/en/issues/)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Before submitting your issue
+        Kindly refer the guideline, [When you submit issue to Floorp!](https://docs.floorp.app/en/others/issue-checklist/)
```

Comment:
It's better to keep the old list, as it lets developers know immediately what actions have been taken. In the case of pointing to the guideline site, what was done is unclear.